pkg/sbom/cyclonedx/unmarshal.go

```diff
@@ -352,6 +353,10 @@ func toPackage(component cdx.Component) (*purl.PackageURL, *ftypes.Package, erro
 	// so we have to use an original package name
 	pkg.Name = getPackageName(p.Type, pkg.Name, component)
 	pkg.Ref = component.BOMRef
+	pkgIdentifier, err := ftypes.NewPkgIdentifier(pkg.Ref)
+	if err == nil {
+		pkg.Identifier = pkgIdentifier
+	}
```

Comment:
I don't UUID are valid to identify known vulnerabilities, at least I mainly find references to use pURL, CPE or SWID:

- https://owasp.org/www-project-web-security-testing-guide/latest/5-Reporting/02-Naming_Schemes
- https://cyclonedx.org/use-cases/#known-vulnerabilities

Reply:
But we also add `PkgIdentifier` for `Package` structure.
This makes sense for users using `trivy fs -f json --list-all-pkgs` command.

Reply:
I understand the use of the UUID within the SBOM doc since you can reference it but it doesn't add much value outside it, right? I think we should omit it.

Reply:
okay, that makes sense.
If we need it, we can always add it.